install.rb

```diff
@@ -108,9 +108,12 @@ def do_man(man, strip = 'man/')
     FileUtils.makedirs(om, {:mode => 0755, :verbose => true})
     FileUtils.chmod(0755, om)
     FileUtils.install(mf, omf, {:mode => 0644, :preserve => true, :verbose => true})
-    gzip = %x{which gzip}
-    gzip.chomp!
-    %x{#{gzip} -f #{omf}}
+    # Solaris does not support gzipped man pages
+    unless Facter["os"].value["family"] == "Solaris"
```

Comment:
Would you think to add a --[no]compress-man or similar option?

I can very easily modify my invocation of install.rb to use that option but it would seem like changing my use of Facter to $operatingsystem would retain the default behavior if --no-check-prereqs is passed.